src/CatalogService/CatalogService.Application/CategoryService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CatalogService.Domain.Entities;
+using CatalogService.Persistence.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace CatalogService.Application
+{
+    public class CategoryService
+    {
+        private readonly IRepository<Category> _categoryRepository;
```

Comment:
Services are more about onion architecture which is quite similar to  the clean architecture. But clean architecture should have "use-cases" which are more atomic than services (services incapsulates many "use-cases"). This video can help you to see the difference between these 2 approaches https://www.youtube.com/watch?v=KqWNtCpjUi8&ab_channel=MilanJovanovi%C4%87

Reply:
I have watched the video and converted the project to Clean Architecture. Thanks!